QtCollider/widgets/QcLevelIndicator.h

```diff
@@ -61,6 +77,8 @@ private Q_SLOTS:
   bool _drawPeak;
   int _ticks;
   int _majorTicks;
+	float _stepWidth;
+	int _style;
```

Comment:
is it possible to convert _style from int to an enum? that will make the code a bit more readable/expressive.